language/Portuguese_Brazil/strings.po

```diff
@@ -1,23 +1,23 @@
-# Brazilian Portuguese translations for MAME Project.
-# Traduções para o Português do Brasil para o Projeto MAME.
-# Copyright (C) 1997-2021  MAMEDev and contributors
+# Brazilian Portuguese translation for the MAME Project.
+# Tradução pro Português Brasileiro para o Projeto MAME.
+# Copyright (C) 1997-2019  MAMEDev and contributors
```

Comment:
The date here should be updated to 2022, not dropped back to 2019, as this has been updated this year.